If you are using glibc 1.99 with Linux (such as current Red Hat for Alpha),
apply this patch in the netkit-base-0.10 directory by doing

	patch -p1 < oldglibc.patch

--- netkit-base-0.10-alpha/ping/ping.c	Wed May 14 09:35:14 1997
+++ netkit-base-0.10/ping/ping.c	Wed May 14 09:36:21 1997
@@ -89,38 +89,8 @@
 #define SAFE_TO_DROP_ROOT
 #endif
 
-#if defined(__GLIBC__) && (__GLIBC__ >= 2)
-#define icmphdr			icmp
-#define ICMP_DEST_UNREACH	ICMP_UNREACH
-#define ICMP_NET_UNREACH	ICMP_UNREACH_NET
-#define ICMP_HOST_UNREACH	ICMP_UNREACH_HOST
-#define ICMP_PORT_UNREACH	ICMP_UNREACH_PORT
-#define ICMP_PROT_UNREACH	ICMP_UNREACH_PROTOCOL
-#define ICMP_FRAG_NEEDED	ICMP_UNREACH_NEEDFRAG
-#define ICMP_SR_FAILED		ICMP_UNREACH_SRCFAIL
-#define ICMP_NET_UNKNOWN	ICMP_UNREACH_NET_UNKNOWN
-#define ICMP_HOST_UNKNOWN	ICMP_UNREACH_HOST_UNKNOWN
-#define ICMP_HOST_ISOLATED	ICMP_UNREACH_ISOLATED
-#define ICMP_NET_UNR_TOS	ICMP_UNREACH_TOSNET
-#define ICMP_HOST_UNR_TOS	ICMP_UNREACH_TOSHOST
-#define ICMP_SOURCE_QUENCH	ICMP_SOURCEQUENCH
-#define ICMP_REDIR_NET		ICMP_REDIRECT_NET
-#define ICMP_REDIR_HOST		ICMP_REDIRECT_HOST
-#define ICMP_REDIR_NETTOS	ICMP_REDIRECT_TOSNET
-#define ICMP_REDIR_HOSTTOS	ICMP_REDIRECT_TOSHOST
-#define ICMP_TIME_EXCEEDED	ICMP_TIMXCEED
-#define ICMP_EXC_TTL		ICMP_TIMXCEED_INTRANS
-#define ICMP_EXC_FRAGTIME	ICMP_TIMXCEED_REASS
-#define	ICMP_PARAMETERPROB	ICMP_PARAMPROB
-#define ICMP_TIMESTAMP		ICMP_TSTAMP
-#define ICMP_TIMESTAMPREPLY	ICMP_TSTAMPREPLY
-#define ICMP_INFO_REQUEST	ICMP_IREQ
-#define ICMP_INFO_REPLY		ICMP_IREQREPLY
-#else
 #define ICMP_MINLEN	28
 #define inet_ntoa(x) inet_ntoa(*((struct in_addr *)&(x)))
-#endif
-
 
 #define	DEFDATALEN	(64 - 8)	/* default data length */
 #define	MAXIPLEN	60
@@ -544,14 +514,12 @@
 	}
 }
 
-#if !defined(__GLIBC__) || (__GLIBC__ < 2)
 #define icmp_type type
 #define icmp_code code
 #define icmp_cksum checksum
 #define icmp_id un.echo.id
 #define icmp_seq un.echo.sequence
 #define icmp_gwaddr un.gateway
-#endif /* __GLIBC__ */
 
 #define ip_hl ihl
 #define ip_v version
